.gitignore

```diff
@@ -0,0 +1,3 @@
+sessions/
+oka.yaml
+bin/
```

Comment:
There's no bin directory created from this repository but there's a build directory.

```suggestion
build/
```